src/30_Advanced/How_to_create_interactive_AMP_pages%3F.html

```diff
@@ -0,0 +1,78 @@
+<!--
+  #### Introduction
+
+  This samples demonstrates how to embed interactive charts into AMP files.
+
+  To achieve its performance, AMP HTML doesn't allow custom Javascript. This doesn't mean that you can't created visually rich and interactive websites with AMP.
+
+  Google Trends recently published [live search trends](https://www.google.com/trends/oscars-2016-in-google-search/index.html) about the Oscars. These gives a fascinating inside into what the billions of searches that take place each day on Google tell us about the nominated actors and movies. Part of the articles are a number of interactive and live updating graphics.
+
+  This sample demonstrates how we can embed these into AMP files.
+-->
+<!-- -->
```

Comment:
This way the docs are shown before the <html> tag. It's a hack I'm using to
place the introduction before the code, e.g. here:
https://ampbyexample.com/introduction/hello_world/. It probably makes sense
to make this more explicit in the future.

On Mon, Mar 21, 2016 at 9:32 PM Chiara Chiappini <notifications@github.com>
wrote:

> In src/30_Advanced/How_to_create_interactive_AMP_pages%3F.html
> <https://github.com/ampproject/amp-by-example/pull/108#discussion_r56901497>
> :
>
> > @@ -0,0 +1,78 @@
> > +<!--
> > +  #### Introduction
> > +
> > +  This samples demonstrates how to embed interactive charts into AMP files.
> > +
> > +  To achieve its performance, AMP HTML doesn't allow custom Javascript. This doesn't mean that you can't created visually rich and interactive websites with AMP.
> > +
> > +  Google Trends recently published [live search trends](https://www.google.com/trends/oscars-2016-in-google-search/index.html) about the Oscars. These gives a fascinating inside into what the billions of searches that take place each day on Google tell us about the nominated actors and movies. Part of the articles are a number of interactive and live updating graphics.
> > +
> > +  This sample demonstrates how we can embed these into AMP files.
> > +-->
> > +<!-- -->
>
> can this <!-- --> be removed?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ampproject/amp-by-example/pull/108/files/5fc669bd56348720ea7a086489cd236a80fc105b#r56901497>
>